docs/media/S4_template.ecf

```diff
@@ -31,12 +31,13 @@ fill_value      mask    # Either the string 'mask' to mask the outlier values (r
 
 # Limb-darkening parameters needed to compute exotic-ld
 compute_ld      True
-inst_filter     Prism #filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
-metallicity     0.1 #metallicity of the star 
-teff            6000 #effective temperature of the star in K
-logg            4.0 #surface gravity in log g
-exotic_ld_direc /home/User/exotic-ld_data/ #directory for ancillary files for exotic-ld
-exotic_ld_grid  3D #1D or 3D model grid
+inst_filter     prism   # Filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
+metallicity     0.1     # Metallicity of the star 
+teff            6000    # Effective temperature of the star in K
+logg            4.0     # Surface gravity in log g
+exotic_ld_direc /home/User/exotic-ld_data/ #directory for ancillary files for exotic-ld, download from: https://zenodo.org/record/6344946
+exotic_ld_grid  3D      # 1D or 3D model grid
+exotic_ld_file  /home/User/exotic-ld_data/Custom_throughput_file.csv # Custom throughput file, for examples see Eureka/demos/JWST/Custom_throughput_files
```

Comment:
There is a conflict here with other ECFs - I believe this should not have the `/home/User/exotic-ld_data/` part of the path

Reply:
I only changed the exotic_ld_file folder and name in the demos/JWST to link to the custom throughput folder in there. The other should have the same generic name as here. Would you prefer it a different way?

Reply:
Leaving it as a fully qualified path is fine with me - you'll just need to edit the file in the demos folder